packages/maskbook/src/plugins/Airdrop/constants.ts

```diff
@@ -6,6 +6,9 @@ export const PLUGIN_NAME = 'Airdrop'
 export const PLUGIN_ICON = '🪂'
 export const PLUGIN_DESCRIPTION = ''
 
+/**
+ * @deprecated Use @masknet/constants
```

Comment:
IMO plugins should host their own constants, then it's easier to split those plugins into a separate package.

Reply:
Use a package for constants will help to reduce duplicate code.

Actually, we have many things that need to be separated from the mono-repo if we are planning to split plugins into separate packages.